examples/perfect_catke_calibration.jl

```diff
@@ -57,9 +57,9 @@ observations = OneDimensionalTimeSeries(data_path, field_names=(:u, :v, :b, :e),
 
 fig = Figure()
 
-ax_b = Axis(fig[1, 1], xlabel = "Buoyancy [m s⁻²]", ylabel = "z [m]")
-ax_u = Axis(fig[1, 2], xlabel = "Velocities [m s⁻¹]", ylabel = "z [m]")
-ax_e = Axis(fig[1, 3], xlabel = "Turbulent kinetic energy [m² s⁻²]", ylabel = "z [m]")
+ax_b = Axis(fig[1, 1], xlabel = "Buoyancy\n[10⁻⁴ m s⁻²]", ylabel = "z [m]")
```

Comment:
noooo

Reply:
I don't like this

Reply:
well, have you seen how the figure looks without this? the x-ticks are a big huge mess...

Reply:
feel free to do something else!

Reply:
hmmm

Reply:
I'll mark this as resolved. It's aesthetic... I just had to do something to make the xticklabels visible. But we can redraw the figures in a way that this doesn't need to happen in a new PR!